tests/testutils/data/init_hook.rc

```diff
@@ -0,0 +1,3 @@
+[MASTER]
+
+init-hook=raise RuntimeError
```

Comment:
Huhu, that's clever.

Reply:
Not actually my idea, found in the tests for `init-hook` as a command line argument. Works quite well though 😄